public/docs/advanced/truffle-with-metamask.md

```diff
@@ -88,12 +88,12 @@ Click the account icon in the upper-right to create new accounts, the first 10 o
 
 ## Using MetaMask with Truffle Develop
 
-Truffle Develop is a command-line application that runs a temporary blockchain that is also used for testing purposes. It runs on `localhost:9545`.
+Truffle Develop is a command-line application that runs a temporary blockchain that is also used for testing purposes. It runs on `127.0.0.1:9545`.
 
 Using MetaMask with Truffle Develop is very similar to that of Ganache. There are only a few notable differences:
 
 * The mnemonic is always `candy maple cake sugar pudding cream honey rich smooth crumble sweet treat` and can't be changed.
-* Truffle Develop runs by default on `localhost:9545`, so you'll want to edit the above web3 code to say:
+* Truffle Develop runs by default on `127.0.0.1:9545`, so you'll want to edit the above web3 code to say:
```

Comment:
Technically this change is more correct, but I think it's more canonical & reads better to leave this one as-is

Reply:
Actually, no - scratch that - don't want to pass the ambiguity on... If anything, we might want to clarify the importance of using `127.0.0.1` instead of `localhost` here...